### PR TITLE
Implemented Gradle Build For Galasa-Comms-Manager

### PR DESCRIPTION
**Pull request by matthewchivers**
Implemented gradle build for:
* galasa-managers-comms-parent
    * dev.galasa.ipnetwork.manager
    * dev.galasa.http.manager
    * dev.galasa.http.manager.ivt

### CODE REVIEW

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Bundle-Name: HTTP Manager
+Export-Package: !dev.galasa.http.internal*;dev.galasa.http*
+Import-Package: com.google.gson,\
+    dev.galasa,\
+    dev.galasa.framework.spi,\
+    dev.galasa.framework.spi.language,\
+    dev.galasa.http.spi,\
+    javax.net.ssl,\
+    javax.validation.constraints;resolution:=optional,\
+    javax.xml.bind,\
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd:11]: jakarta.xml.bind

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    javax.xml.bind.annotation,\
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd:12]: jakarta.xml.bind.annotation

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    javax.xml.parsers,\
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/bnd.bnd:13]: dont change

**Review by Michael-Baylis — CHANGES_REQUESTED** [review]

**Comment by matthewchivers** [comment]
Changes have been made as requested.
Just working on getting a passing integration test.

**Comment by matthewchivers** [comment]
Integration test passing 👍

**Review by Michael-Baylis — APPROVED** [review]

**Pull request merged**